docs/_sass/minima/_base.scss

```diff
@@ -288,7 +288,7 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "Academy Assist";
```

Comment:
And also thanks for fixing the header problem for this too!

Reply:
Nice catch!